plugins/brew/brew.plugin.zsh

```diff
@@ -4,6 +4,7 @@ alias brewsp='brew list --pinned'
 alias bubo='brew update && brew outdated'
 alias bubc='brew upgrade && brew cleanup'
 alias bubu='bubo && bubc'
+alias brewsl='brew services list'
```

Comment:
How about `bsl` to match the schema? It's [not taken by any other plugin](https://github.com/robbyrussell/oh-my-zsh/search?q=bsl&unscoped_q=bsl) so far :-)